src/lib/EEPROM/elrs_eeprom.cpp

```diff
@@ -1,38 +1,51 @@
 #include "elrs_eeprom.h"
+#include <Arduino.h>
 
 #ifdef PLATFORM_STM32
-extEEPROM EEPROM(kbits_2, 1, 1);
+    extEEPROM EEPROM(kbits_2, 1, 1, 0x51);
 #endif
 
 void
 ELRS_EEPROM::Begin()
 {
 #ifdef PLATFORM_STM32
-    Wire.setSDA(GPIO_PIN_SDA); // set is needed or it wont work :/
-    Wire.setSCL(GPIO_PIN_SCK);
-    Wire.begin();
```

Comment:
We may need to keep set and begin.  The r9 receivers do not use the DAC lib where Wire is initialised.  I dont think there is any problem with calling begin twice, but needs to be checked.